README.md

```diff
@@ -10,21 +10,134 @@ An unofficial, experimental Python client library for the [TransferWise API](htt
 # Within your project directory
 pip install pywisetransfer
 ```
+After installation, you should be able to import the package.
+
+```python
+import pywisetransfer
+```
```

Comment:
Let's omit this; the package is a simple one, and provided that the installation process succeeds, this should too.

We should consider including it in complete code examples, however also avoid too much repetition.